packages/account/src/Components/poinc/status/limited/index.js

```diff
@@ -0,0 +1,3 @@
+import { PoincLimited } from './poinc-limited.jsx';
```

Comment:
Same here. Because it is already a file under `/poinc/` directory.
```suggestion
import { PoincLimited } from './limited.jsx';
```

Reply:
done :+1: